modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -100,7 +100,21 @@ private void performTestForScriptedSchema(String jdbcUrl) throws SQLException {
                 assertEquals("A basic SELECT query succeeds where the schema has been applied from a script", "hello world", resultSetString);
                 return true;
             });
-
+            
+            result = new QueryRunner(dataSource).query("select CURRENT_USER()", rs -> {
+              rs.next();
+              String resultUser = rs.getString(1);
+              assertEquals("User from query param is created.", "someuser@%", resultUser);
+              return true;
+          });
+            
+            result = new QueryRunner(dataSource).query("SELECT DATABASE()", rs -> {
+              rs.next();
+              String resultDB = rs.getString(1);
+              assertEquals("Database name from URL String is used.", "databasename", resultDB);
+              return true;
+          });
+           
```

Comment:
This is just a trivial comment, but please could you reformat using 4 spaces for indentation? IntelliJ/Eclipse defaults ought to be reasonably close. I'll add automatic formatting soon...

Reply:
Sure, I'll do that. Thanks for reviewing!

Reply:
I have updated formatting in recent commit.